### ISSUE TITLE
Improve handling of feedback when connection has gone in timeout

### ISSUE THREAD

**Issue by apupier** [eclipse-che/che#13168]
### Description

I had a workspace working.
I left it for few hours then when I came back, several actions were not responding.

it would be nice to have an error message explaining what is happening and/or redirecting to login page./

### Reproduction Steps

- open a Che 7 workspace
- go away for several hours
- try to use again the workspace, for instance completion is blocked on "loading"

**OS and version:**    

Chrome
windows 10
che.openshift.io
Che 7 stack

**Diagnostics:** 

In console log, there is a loop with errors:
```
Event {isTrusted: true, type: "error", target: WebSocket, currentTarget: WebSocket, eventPhase: 2, …}
error (async)
(anonymous) @ vendors.c110bfbb4aa2cd71c473.js:1
(anonymous) @ vendors.c110bfbb4aa2cd71c473.js:1
w @ vendors.c110bfbb4aa2cd71c473.js:1
setTimeout (async)
b @ vendors.c110bfbb4aa2cd71c473.js:1
vendors.c110bfbb4aa2cd71c473.js:1
```

```
Error: Connection timeout
    at vendors.c110bfbb4aa2cd71c473.js:1
(anonymous) @ vendors.c110bfbb4aa2cd71c473.js:1
setTimeout (async)
y @ vendors.c110bfbb4aa2cd71c473.js:1
(anonymous) @ vendors.c110bfbb4aa2cd71c473.js:1
setTimeout (async)
w @ vendors.c110bfbb4aa2cd71c473.js:1
setTimeout (async)
b @ vendors.c110bfbb4aa2cd71c473.js:1
vendors.c110bfbb4aa2cd71c473.js:1
```

```
WebSocket connection to 'wss://routerjnakrv5-apupier-demo-che.9a6d.starter-us-east-1a.openshiftapps.com/connect?token=XXXXXXX' failed: Error during WebSocket handshake: Unexpected response code: 503
```
